### PR TITLE
Adds a cool boot sequence to nanosuit and rids morphine injection

### PR DESCRIPTION
**Pull request by yoyobatty**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs )

:cl: YoYoBatty
add: Nanosuit has a bootup sequence, sensors come online as it loads in.
add: Nanosuit has a "revive" sequence in emp shutdown.
del: Removed nanosuit morphine injection from limb damage reaction.
fix: Var refs in nanosuit were inconsistent in some cases.
fix: Nanosuit code has strange behaviour that has now been fixed.

/:cl:

[why]: So apparently morphine was making users fall asleep whenever they had any sort of limb trauma or whatever, it was too much of a hassle to fix and caused a temporary speed boost which made the nanosuit a bit more powerful than it needed to be while at the same time fucking over the user. Long story short, I removed the injection.
The dank ass boot sequence is inspired by old computers, you will notice that all the sensors appear offline and will actually come on, section by section as the messages appear. I think that's pretty cool. Let me know what you think.

https://imgur.com/a/1iBHlOg

### CODE REVIEW

**Comment by Manly-Dwarf** [comment]
The morphine injection needed to be removed because it was just a flat out bad idea that punished the user for taking any damage at all.
Nanosuit as a whole still needs more work.

**Comment by yoyobatty** [comment]
Yeah, I agree 100%. I'm really committed to making this as good as I possibly can make it so feel free to pitch ideas and changes.

**Comment by bubberducky2** [comment]
Epic

File: hippiestation/code/modules/clothing/spacesuits/nanosuit.dm [hunk 1]
```diff
@@ -647,8 +650,41 @@
 		U.add_trait(TRAIT_NODISMEMBER, "Nanosuit")
 		if(help_verb)
 			U.verbs += help_verb
+		bootSequence()
 	..()
 
+/obj/item/clothing/suit/space/hardsuit/nano/proc/bootSequence()
+	helmet.display_visor_message("Crynet - BIOS v1.32 Syndicate Systems")
```
Comment (ghost) [inline hippiestation/code/modules/clothing/spacesuits/nanosuit.dm:667]: UEFI*

Comment (yoyobatty) [reply hippiestation/code/modules/clothing/spacesuits/nanosuit.dm:667]: >Unified Extensible Firmware Interface? I prefer basic input/output system, nerd

**Comment by Eagle-Eyess** [comment]
Sure +1.

**Pull request closed without merging**